spec/rspec/matchers/built_in/yield_spec.rb

```diff
@@ -67,6 +67,16 @@ def each_arg(*args, &block)
       expect(val).to be_nil
     end
 
+    it 'raises an error if given an invalid count argument' do
+      expect { yield_control.exactly('2') }.to raise_error(ArgumentError)
+      expect { yield_control.at_least(:trice_with_typo) }.to raise_error(ArgumentError)
+      expect { yield_control.at_most(nil) }.to raise_error(ArgumentError)
+    end
+
+    it 'forbids multiple calls to count constraints' do
+      expect { yield_control.at_least(1).at_most(4).times }.to raise_error(/multiple/i)
```

Comment:
I would expect this to work, are you saying it doesn't or do you want to change this to not work?

Reply:
Currently, the call to `at_most(4)` silently overwrites the constraint set by `at_least(1)`. This (amended) PR simply raises a `NotImplementedError`. If it is deemed necessary to support ranges, it could be implemented.